deployment/ccip/changeset/v1_5_1/cs_configure_token_pools.go

```diff
@@ -23,6 +23,12 @@ import (
 
 var _ deployment.ChangeSet[ConfigureTokenPoolContractsConfig] = ConfigureTokenPoolContractsChangeset
 
+// AllowedTokenSymbolOverrides is a list of token pool symbols that are allowed to be overridden.
+var AllowedTokenSymbolOverrides = map[changeset.TokenSymbol]changeset.TokenSymbol{
```

Comment:
We can remove this, I think it is okay to trust that we will only use the override when confident